server/src/test/java/com/objectcomputing/checkins/services/settings/SettingsControllerTest.java

```diff
@@ -283,4 +276,15 @@ void testDELETESettingNoPermission() {
         assertEquals(HttpStatus.UNAUTHORIZED,responseException.getStatus());
 
     }
+
```

Comment:
For coverage, is it worth testing the UPDATE with an unknown (but valid) setting name, as in

```suggestion

    @Test
    void testPUTValidDTOButUnknownName() {
        final MemberProfile lucy = getMemberProfileRepository().save(mkMemberProfile("Lucy"));

        SettingsDTO updatedSetting = new SettingsDTO();
        updatedSetting.setName(SettingOption.LOGO_URL.toString());
        updatedSetting.setValue("Missing");

        final HttpRequest<SettingsDTO> request = HttpRequest.
                PUT("/", updatedSetting).basicAuth(lucy.getWorkEmail(),ADMIN_ROLE);

        HttpClientResponseException responseException = assertThrows(HttpClientResponseException.class,
                () -> client.toBlocking().exchange(request, Map.class));

        assertEquals(HttpStatus.NOT_FOUND, responseException.getStatus());
        assertEquals("Setting with name %s not found.".formatted(SettingOption.LOGO_URL.toString()), responseException.getMessage());
    }
```

Reply:
> For coverage, is it worth testing the UPDATE with an unknown (but valid) setting name, as in

Good catch. Updated.